### PR TITLE
documentation added for issue #3685

### PR DESCRIPTION
**Pull request by tibistibi**

### CODE REVIEW

**Comment by deepu105** [comment]
@MathieuAA could you please see if this is ok?

File: pages/jdl.md [hunk 1]
```diff
@@ -210,6 +210,22 @@ You can do the same with the JDL:
 The keywords `dto`, `paginate`, `service` and `with` were added to the grammar to support these changes.
 If a wrong option is specified, JDL will inform you of that with a nice, red message and will just ignore it so as not to corrupt JHipster's JSON files.
 
+#### Service option
+
+No services specified will create a resource class which will call the repository interface directly. This is the default and simplest option, see A.
```
Comment (MathieuAA) [inline pages/jdl.md]: `Not specifying a service` instead of `No services specified` seems prettier?

File: pages/jdl.md [hunk 2]
```diff
@@ -210,6 +210,22 @@ You can do the same with the JDL:
...
+Service with serviceClass (see B) will make the resource call the service class which will call the repository interface. Service with serviceImpl (see C) will make a service interface which will be used by the resource class. The interface is implemented by an impl class which will call the repository interface.
+
+Use no service if not sure it's the simplest option and good for CRUD. Use service with a Class if you will have a lot of business logic which will use multple repository's making it ideal for a service class. Jhipster's are not a fan of unnecessary Interfaces but if you like them go for service with impl.
```
Comment (MathieuAA) [inline pages/jdl.md]: `Use no service if not sure,`, added the comma.

Comment (MathieuAA) [inline pages/jdl.md]: `Jhipster's are not a fan of unnecessary Interfaces`, `JHipsters... interfaces`

**Comment by MathieuAA** [comment]
Added a couple of wording things, but it's good to go!

**Comment by tibistibi** [comment]
thanks!

happy to improve :smile:

**Pull request merged**